Adjust_dataset.py

```diff
@@ -0,0 +1,43 @@
+#Script changing labels of emotions to 1 - smile, 0 - no smile and saving changed dataset to new csv file
+
+import pandas as pd
+
+data_set = pd.read_csv("fer2013.csv")
```

Comment:
Please pass path to csv from command line argument. Here some help https://docs.python.org/3/library/argparse.html